pcsx2/gui/FrameForGS.cpp

```diff
@@ -438,7 +438,7 @@ GSFrame::~GSFrame() throw()
 void GSFrame::OnCloseWindow(wxCloseEvent& evt)
 {
 	sApp.OnGsFrameClosed( GetId() );
-	Hide();		// and don't close it.
+	CoreThread.Reset(); // reset thread to avoid suspend status on close event.
```

Comment:
Not sure I understand this change. Also, I'm pretty sure that using `Reset` instead of `Hide` could break some things since this is a sensitive behavior of PCSX2.

Wouldn't it be better overall to just suspend if it's running (and recall that it needs to resume) and then resume only if resume is needed?

(I'm not claiming that the code is necessarily bad, just that I'd imagined it would need less changes, and I'm wondering whether all the changes here are actually required in order to fix the issue)